waila/src/lib.rs

```diff
@@ -847,6 +886,25 @@ mod tests {
         )
     }
 
+    #[test]
+    fn parse_fedimint_oob_notes() {
+        let parsed = PaymentParams::from_str(SAMPLE_FEDIMINT_OOB_NOTES).unwrap();
+
+        assert_eq!(parsed.address(), None);
+        assert_eq!(parsed.memo(), None);
+        assert_eq!(parsed.network(), None);
+        assert_eq!(parsed.invoice(), None);
+        assert_eq!(parsed.node_pubkey(), None);
+        assert_eq!(parsed.amount(), Some(Amount::from_sat(10)));
+        // NOTE: (@leonardo) there is not `Eq` implementation for `fedimint-mint-client::OOBNotes`
```

Comment:
added: https://github.com/fedimint/fedimint/pull/4346

Reply:
nice! thx!

Reply:
@benthecarman would be a problem using a different release/commit here in waila when comparing it to mutiny-node ? 🤔

Reply:
We should keep them in sync

Reply:
ok, so we'll need to fix this when bumping the mutiny-node to a new release

Reply:
Yeah that's fine. Hopefully one day it'll all just be on 0.3